tests/NuGet.IndexingTests/NuGetQueryTests.cs

```diff
@@ -8,17 +11,232 @@ namespace NuGet.IndexingTests
+                new BooleanClause(new BooleanQuery { new BooleanClause(new TermQuery(new Term("Owner", "microsoft")), Occur.SHOULD) }, Occur.MUST)
+            };
+
+            // act
+            var actual = NuGetQuery.MakeQuery(queryText);
+
+            // assert
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void EmptyQueryMatchesAllDocuments()
+        {
+            // arrange
+            var queryText = string.Empty;
+            var expected = new MatchAllDocsQuery();
+
+            // act
+            var actual = NuGetQuery.MakeQuery(queryText);
+
+            // assert
+            Assert.Equal(expected, actual);
+        }
+
+        [Fact]
+        public void TreatsUnrecognizedFieldAsAnyField()
+        {
+            // arrange
+            var queryText = "invalid:dot";
+            var expected = new BooleanQuery
+            {
+                new BooleanClause(new BooleanQuery { Clauses = { new BooleanClause(new TermQuery(new Term("Id", "dot")), Occur.SHOULD) }, Boost = 8 }, Occur.SHOULD),
+                new BooleanClause(new BooleanQuery { new BooleanClause(new TermQuery(new Term("ShingledId", "dot")), Occur.SHOULD) }, Occur.SHOULD),
+                new BooleanClause(new BooleanQuery { new BooleanClause(new TermQuery(new Term("TokenizedId", "dot")), Occur.SHOULD) }, Occur.SHOULD),
+                new BooleanClause(new BooleanQuery { new BooleanClause(new TermQuery(new Term("Version", "dot")), Occur.SHOULD) }, Occur.SHOULD),
+                new BooleanClause(new BooleanQuery { new BooleanClause(new TermQuery(new Term("Title", "dot")), Occur.SHOULD) }, Occur.SHOULD),
+                new BooleanClause(new BooleanQuery { new BooleanClause(new TermQuery(new Term("Description", "dot")), Occur.SHOULD) }, Occur.SHOULD),
+                new BooleanClause(new BooleanQuery { new BooleanClause(new TermQuery(new Term("Summary", "dot")), Occur.SHOULD) }, Occur.SHOULD),
+                new BooleanClause(new BooleanQuery { Clauses = { new BooleanClause(new TermQuery(new Term("Tags", "dot")), Occur.SHOULD) }, Boost = 2 }, Occur.SHOULD),
+                new BooleanClause(new BooleanQuery { new BooleanClause(new TermQuery(new Term("Authors", "dot")), Occur.SHOULD) }, Occur.SHOULD),
+                new BooleanClause(new BooleanQuery { new BooleanClause(new TermQuery(new Term("Owner", "dot")), Occur.SHOULD) }, Occur.SHOULD)
+            };
+
+            // act
+            var actual = NuGetQuery.MakeQuery(queryText);
+
+            // assert
+            Assert.Equal(expected, actual);
+        }
+
+        public static IEnumerable<object[]> MakesQueriesWithProperPhrasingData
+        {
+            get
+            {
+                // multiphrase query
+                {
+                    var multiPhraseQuery = new MultiPhraseQuery();
+                    multiPhraseQuery.Add(new[] { new Term("Title", "dotnetzip"), new Term("Title", "dot"), new Term("Title", "dotnet") });
```

Comment:
you want to break long lines, basically you want them to fit in a github review without scrolling